src/input-schemas.json

```diff
@@ -0,0 +1,1028 @@
+      "default": 2000,
+      "description": "Year of investment and operation decisions in the optimization.",
+      "type": "INTEGER"
+    },
+    "to_asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    }
+  },
+  "flow_commission": {
+    "commission_year": {
+      "UoM": null,
+      "constraints": null,
+      "default": 2000,
+      "description": "Year of commissioning",
+      "type": "INTEGER"
+    },
+    "efficiency": {
+      "UoM": "ratio",
+      "constraints": {
+        "minimum": 0
+      },
+      "default": 1,
+      "description": "Efficiency of transfer in `from_asset` -> `to_asset` direction",
+      "type": "DOUBLE"
+    },
+    "fixed_cost": {
+      "UoM": "CUR/MW/year",
+      "constraints": null,
+      "default": null,
+      "description": "Fixed annual cost for the transport asset capacity.",
+      "type": "DOUBLE"
+    },
+    "from_asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "investment_cost": {
+      "UoM": "CUR/MW",
+      "constraints": null,
+      "default": null,
+      "description": "Investment cost for the transport asset capacity.",
+      "type": "DOUBLE"
+    },
+    "investment_limit": {
+      "UoM": "MWh",
+      "constraints": null,
+      "default": null,
+      "description": "Maximum capacity for the transport asset investment. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "to_asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    }
+  },
+  "flow_milestone": {
+    "from_asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "investable": {
+      "UoM": null,
+      "constraints": null,
+      "default": false,
+      "description": "Whether there is an investment variable created for the asset or not.",
+      "type": "BOOLEAN"
+    },
+    "milestone_year": {
+      "UoM": null,
+      "constraints": null,
+      "default": 2000,
+      "description": "Year of investment and operation decisions in the optimization.",
+      "type": "INTEGER"
+    },
+    "to_asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "variable_cost": {
+      "UoM": "CUR/MWh",
+      "constraints": null,
+      "default": 0,
+      "description": "Variable cost for the flow.",
+      "type": "DOUBLE"
+    }
+  },
```

Comment:
Got tired around here. Hope the rest of the file is similar enough to the beginning, and similar comments apply.